### DISCUSSION TITLE
ETPRO Tune Detections missing

### DISCUSSION THREAD

**Discussion by xaar1982** [Security-Onion-Solutions/securityonion#13871] [2.4]
### Version

2.4.100

### Installation Method

Security Onion ISO image

### Description

configuration

### Installation Type

Distributed

### Location

on-prem with Internet access

### Hardware Specs

Exceeds minimum requirements

### CPU

8

### RAM

32

### Storage for /

500 GB

### Storage for /nsm

350 GB

### Network Traffic Collection

other (please provide detail below)

### Network Traffic Speeds

1Gbps to 10Gbps

### Status

Yes, all services on all nodes are running OK

### Salt Status

No, there are no failures

### Logs

No, there are no additional clues

### Detail

Hello, 

I have some problems with tune detection entries. I'm using ETPRO rulesets. I had a lot of tuning entries, that suppressed false postives. I set them up using SOC portal and "Tuning" tab. Now they are all gone....I see information in ruleset history that System created one again ruleset but at the same time it deleted all my entries. Below screenshot. The only action i done was reboot hosts via Grid panel to update OS. Is that proper behaviour ? If yes how can I prevent it in future ?
<img width="1643" alt="Zrzut ekranu 2024-10-28 o 08 46 43" src="https://github.com/user-attachments/assets/6f0e1682-b133-418d-b837-303706dd509d">

### Guidelines

- [X] I have read the discussion guidelines at https://github.com/Security-Onion-Solutions/securityonion/discussions/1720 and assert that I have followed the guidelines.

**Comment by defensivedepth** [comment]
Did you change any other configuration of your system? Airgap? Swap from ETPRO to ETOPEN then back to ETPRO?

**Reply by xaar1982** [reply]
Nope, the only thing I did was reboot machines due OS upgrade from SOC GRID

**Reply by defensivedepth** [reply]
Click on the following, set the timeframe to 2 days within when all this happened, and let me know what types of logs are seeing. 

![image](https://github.com/user-attachments/assets/25a3a706-6b5e-4a71-b8b0-b451ed9a7479)


------

![image](https://github.com/user-attachments/assets/1c6b6f5e-d976-44fd-b6fa-348d198e3fe9)
